src/main/java/attendance/model/Holiday.java

```diff
@@ -0,0 +1,28 @@
+package attendance.model;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+
+public enum Holiday {
+
+    CHRISTMAS(Month.DECEMBER, 25),
+    ;
+
+    private final Month month;
+    private final int date;
+
+    Holiday(Month month, int date) {
+        this.month = month;
+        this.date = date;
+    }
+
+    private boolean isSame(Month month, int date) {
+        return this.month == month && this.date == date;
+    }
+
+    public static boolean isHoliday(LocalDate date) {
+        return Arrays.stream(values())
+                .anyMatch(holiday -> holiday.isSame(date.getMonth(), date.getDayOfMonth()));
+    }
```

Comment:
isSame의 메서드 파라미터로 `LocalDate`를 받도록해서 내부에서 Month, date를 직접 get해서 비교하도록 변경해보면 어떨까요? 

```suggestion
    private boolean isSame(LocalDate localDate) {
        return this.month == localDate.getMonth() && this.date == localDate.getDayOfMonth();
    }

    public static boolean isHoliday(LocalDate date) {
        return Arrays.stream(values())
                .anyMatch(holiday -> holiday.isSame(date));
    }
```

Reply:
확실히 LocalDate로 변경한다면 중복을 줄일 수 있겠네요!! 

여기서 궁금한점이 있는데 매개변수로 위와 같이 필요한 값만 가질지, 아니면 LocalDate와 같이 좀더 범위가 큰 객체로 받을지 고민이 됐었는데 혹시 어떤 기준으로 선택을 하시나요?